tools/dataproc-edge-node/1_create-image.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function err() {
+  echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $@" >&2
```

Comment:
In general it's good to avoid mixing variables inside strings, the preferred way is to use + and quote the variable. Something like   echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: " +  "${@}" >&2

Reply:
Can you provide a citation?
Your example doesn't work. The '+' symbol is printed between the variables along with extra spaces.